src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MetricsEventSource.cs

```diff
@@ -115,40 +115,43 @@ public void CollectionStop(string sessionId, DateTime intervalStartTime, DateTim
             WriteEvent(3, sessionId, intervalStartTime, intervalEndTime);
         }
 
-        [Event(4, Keywords = Keywords.TimeSeriesValues, Version = 1)]
+        [Event(4, Keywords = Keywords.TimeSeriesValues, Version = 2)]
 #if !NET8_0_OR_GREATER
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
                                       Justification = "This calls WriteEvent with all primitive arguments which is safe. Primitives are always serialized properly.")]
 #endif
-        public void CounterRateValuePublished(string sessionId, string meterName, string? meterVersion, string instrumentName, string? unit, string tags, string rate, string value)
+        public void CounterRateValuePublished(string sessionId, string meterName, string? meterVersion, string instrumentName, string? unit, string tags, string rate, string value,
+                                    string instrumentFormattedTags, string instrumentHash, string meterFormattedTags, string meterHash, string formattedScopeHash)
         {
-            WriteEvent(4, sessionId, meterName, meterVersion ?? "", instrumentName, unit ?? "", tags, rate, value);
+            WriteEvent(4, sessionId, meterName, meterVersion ?? "", instrumentName, unit ?? "", tags, rate, value, instrumentFormattedTags, instrumentHash, meterFormattedTags, meterHash, formattedScopeHash);
         }
 
-        [Event(5, Keywords = Keywords.TimeSeriesValues)]
+        [Event(5, Keywords = Keywords.TimeSeriesValues, Version = 2)]
 #if !NET8_0_OR_GREATER
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
                                       Justification = "This calls WriteEvent with all primitive arguments which is safe. Primitives are always serialized properly.")]
 #endif
-        public void GaugeValuePublished(string sessionId, string meterName, string? meterVersion, string instrumentName, string? unit, string tags, string lastValue)
+        public void GaugeValuePublished(string sessionId, string meterName, string? meterVersion, string instrumentName, string? unit, string tags, string lastValue,
+                                    string instrumentFormattedTags, string instrumentHash, string meterFormattedTags, string meterHash, string formattedScopeHash)
```

Comment:
`InstrumentPublished` event has "meterTags" & "instrumentTags" here we have "meterFormattedTags" & "instrumentFormattedTags" are they the same things? If so can we be consistent with the names?

Reply:
done!